### PR TITLE
Define Equipment Create Action

### LINKED ISSUE #6: [1pt] Set up the logic for the create action of the EquipmentsController
- [x] Create the logic for the create action for EquipmentsController in it's create Equipment Interactor
- [x] Create the RSpec test for the create action EquipmentsController.
- [x] Add validation test cases to the model
- [x] Create any helper method in the Equipment model that it may be needed.

### PR DESCRIPTION
**Pull request by JohnFTitor**
# Hi there 👋🏻

Thanks for taking the time to review the project. Here are some things important to note about it:

### Changes implemented in this PR

- [X] Closes #6 
- [X] Add JSONAPI serializer to return a proper response with the image attached
- [X] Establish relation between ActiveStorage and the Equipment

## Final notes 📔 

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

### CODE REVIEW

File: app/models/equipment.rb [hunk 1]
```diff
@@ -1,5 +1,13 @@
 class Equipment < ApplicationRecord
+  has_one_attached :image
```
Comment (jose-Abel) [inline app/models/equipment.rb:2]: It's going to be just one image or many images?

**Review by jose-Abel — CHANGES_REQUESTED** [review]
# Hi @JohnFTitor 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there## Highlights## Required Changes :recycle:
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is just a personal suggestion that I think can make your code better. I'm open to discussing your opinions around them but are not required so we can continue the development of the project.

Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
------
This is a feature teammate review. Remember that everything said here is intended for both of us to learn and grow as Software Developers :computer:  and we may have different approaches or perspectives. That being said, don't hesitate to comment if you disagree with one required change. We can talk everything through.

File: app/models/equipment.rb (hunk 1 above)
Comment (JohnFTitor) [reply app/models/equipment.rb:2]: We checked the design again and it's just one image @jose-Abel

Comment (jose-Abel) [reply app/models/equipment.rb:2]: Ok cool, thanks, so is approved!

Comment (JohnFTitor) [reply app/models/equipment.rb:2]: Great! Thanks

**Pull request merged**